app/src/main/java/com/example/whatshouldiread/customclasses/Book.java

```diff
@@ -0,0 +1,142 @@
+package com.example.whatshouldiread.customclasses;
+
+/*
+ * Class that stores information for a particular book. Allows for links to amazon for the particular
+ * book be created through a function, rather than hardcoding.
+ */
+
```

Comment:
@petromw Updated the Javadocs for the files based on the comments from the original pull request. Please resolve this comment when you're okay with these changes.